content/docs/reference/spec/migration/platform-api-0.7-0.8.md

```diff
@@ -0,0 +1,48 @@
++++
+title="Platform API 0.7 -> 0.8"
++++
+
+<!--more-->
+
+This guide is most relevant to platform operators.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/platform%2Fv0.8) for platform API 0.8 for the full list of changes and further details.
+
+## Platform Operator
+
+### New standardized SBOM format
+
+Buildpacks implementing Buildpack API 0.7+ may output write Software Bill of Materials (SBOM) files describing build- or run-time dependencies. These files must use the `application/vnd.cyclonedx+json`, `application/spdx+json`, or `application/vnd.syft+json` media types (a buildpack may output SBOM files in multiple formats). Files may be written to the following locations:
+                                                      
+* `<layers>/<buildpack-id>/<layer>.sbom.<ext>` - for describing dependencies associated with a layer
+* `<layers>/<buildpack-id>/launch.sbom.<ext>` - for describing run-time dependencies not associated with a layer
+* `<layers>/<buildpack-id>/build.sbom.<ext>` - for describing build-time dependencies not associated with a layer
```

Comment:
I noticed that there's no mention of `<layers>/<buildpack-id>/cache.sbom.<ext>`. Not sure if that was intentional.

Reply:
Right, that is just a lifecycle implementation detail and is not spec'd.